gammapy/data/tests/test_data_store.py

```diff
@@ -58,6 +58,15 @@ def test_DataStore_load(data_manager):
     events = data_store.load(obs_id=23037, filetype='events')
     assert isinstance(events, EventList)
 
+#@pytest.mark.xfail
+@requires_data('hess')
+def test_DataStore_load_all():
+    """Test loading data and IRF files via the DataStore"""
+    dir = str(gammapy_extra.dir) + '/datasets/hess-crab4'
+    data_store = DataStore.from_dir(dir)
+    event_lists = data_store.load_all(filetype='events')
+    assert event_lists[0]['ENERGY'][0] == 1.1156039
```

Comment:
The == assert doesn't work for float number comparisons.
Use this instead:
https://gammapy.readthedocs.org/en/latest/development/howto.html#assert-convention